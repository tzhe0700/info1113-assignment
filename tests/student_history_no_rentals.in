ADD STUDENT Harry Potter
STUDENT HISTORY 100000
EXIT
