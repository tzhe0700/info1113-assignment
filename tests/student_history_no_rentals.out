user: Success.

user: No rental history for student.

user: Ending Archive process.
